config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-  
+  root "users#index"
+
+  get "/users", to: "users#index"
+  get "/users/:id/posts", to: "users#posts"
+  get "/users/:id", to: "users#show"
```

Comment:
- [ ] OPTIONAL:  As a tip, here is a shorthand version of what you did:

```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
Thanks @Meltrust
I will make changes accordingly.